tabs/tab_survival.py

```
@@ -578,28 +613,29 @@ def out_landmark_result():
-        """Render landmark analysis plot."""
         res = landmark_result.get()
         return res['fig'] if res else None
 
     @render.data_frame
     def out_landmark_table():
-        """Render landmark analysis table."""
         res = landmark_result.get()
         return render.DataGrid(res['stats']) if res else None
 
     @render.download(filename="landmark_report.html")
     def btn_dl_landmark():
```

Ruff (0.14.10)

Missing return type annotation for private function `btn_dl_landmark`

(ANN202)